{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": []
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "info@cert.vde.com",
      "ID": "CVE-2018-25108"
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An unauthenticated remote attacker can cause a DoS in the controller due toÂ uncontrolled resource consumption."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-770"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "https://cert.vde.com/en/advisories/VDE-2018-013",
          "refsource": "",
          "tags": [],
          "url": "https://cert.vde.com/en/advisories/VDE-2018-013"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "NETWORK",
        "availabilityImpact": "HIGH",
        "baseScore": 7.5,
        "baseSeverity": "HIGH",
        "confidentialityImpact": "NONE",
        "integrityImpact": "NONE",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
        "version": "3.1"
      },
      "exploitabilityScore": 3.9,
      "impactScore": 3.6
    }
  },
  "lastModifiedDate": "2025-01-16T11:15Z",
  "publishedDate": "2025-01-16T11:15Z"
}